src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = 0;
+  const dailyRentalCost = 40;
+  const threeDayDiscount = 20;
+  const sevenDayDiscount = 50;
+
+  for (let i = 0; i < days; i++) {
+    totalCost += dailyRentalCost;
+  }
```

Comment:
The loop for calculating the total cost can be replaced with `totalCost = days * dailyRentalCost;` to improve efficiency.